media-libs/libshumate/libshumate-1.0.1.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+VALA_USE_DEPEND="vapigen"
+
+inherit gnome.org meson vala
+
+DESCRIPTION="Shumate is a GTK toolkit providing widgets for embedded maps."
+HOMEPAGE="https://wiki.gnome.org/Projects/libshumate"
+
+SLOT="0"
+LICENSE="LGPL-2.1+"
+KEYWORDS="~amd64"
+RESTRICT="test"
+
+IUSE="introspection gtk-doc vector-renderer vala"
```

Comment:
Alphabetize.

Also, if `vector-renderer` is experimental I don't think we should offer it at least initially, unless there's a good reason.